snapshot/src/main/java/io/camunda/zeebe/snapshots/PersistedSnapshotStore.java

```diff
@@ -78,4 +78,14 @@
   ActorFuture<Void> delete();
 
   Path getPath();
+
+  /**
+   * Copy snapshot to the given directory
+   *
+   * @param snapshot the snapshot to be copied
+   * @param targetDirectory the directory to which the snapshot files will be copied
+   * @return future which will be completed exceptionally if the snapshot was not copied
+   *     successfully
+   */
+  ActorFuture<Void> copySnapshot(PersistedSnapshot snapshot, Path targetDirectory);
```

Comment:
:thought_balloon: First I was a bit confused why this need to be on the public api :see_no_evil: But later I realized that this will be used for the case when we need to copy a snapshot to the runtime folder. Sorry seem to be too early for me :laughing: . Do you think it might make sense to have it on the Snapshot interface instead of the store? :thinking:

Reply:
I don't have a strong opinion here. Both makes sense to me. Would you rather prefer to have it in `PersisteSnapshot`?